src/app/components/Copyright/index.stories.tsx

```diff
@@ -1,14 +1,13 @@
 import React from 'react';
-import { withServicesKnob } from '#psammead/psammead-storybook-helpers/src';
-import { withKnobs } from '@storybook/addon-knobs';
 import services from '#server/utilities/serviceConfigs';
+import withServicesDecorator from '#storybook/withServicesDecorator';
 import { ServiceContext } from '../../contexts/ServiceContext';
 import CopyrightContainer from '.';
-import { StoryProps } from '../../models/types/storybook';
+import { StoryProps, StoryArgs } from '../../models/types/storybook';
 import { ServiceConfig } from '../../models/types/serviceConfig';
 
 // eslint-disable-next-line react/prop-types
-const Component = ({ service, variant }: StoryProps) => {
+const Component = (_: StoryArgs, { service, variant }: StoryProps) => {
```

Comment:
Can we get rid of this lint override, or is it still required?

Reply:
Not needed now, will remove.